content/rdi/quickstart/write-behind-guide.md

```diff
@@ -1,7 +1,7 @@
 ---
-Title: Write-behind quickstart
-linkTitle: Write-behind
-description: Get started creating a write-behind pipeline
+Title: Write-behind & Read-Through quickstart
+linkTitle: Write-behind & Read-Through
+description: Get started creating a pipeline for redis synchronization with a downstream database (write-behind, read-through, write-through (pending))
```

Comment:
I'd remove write-through completely. Pending is not that clear that this feature is yet not supported, IMO.

Reply:
done